app/controllers/hyrax/citations_controller.rb

```diff
@@ -3,7 +3,6 @@ module Hyrax
   class CitationsController < ApplicationController
     include WorksControllerBehavior
     include Breadcrumbs
-    include SingularSubresourceController
```

Comment:
@eliotjordan and I looked at this together to determine why it is ok to remove this.

From Eliot...
> SingularSubresourceController is bound tightly to ActiveFedorda and removing the controller mix-in enables citations for Valkyrie and non-Valkyrie works.
>
> SingularSubresourceController finds the work using the Hyrax::WorkRelation class: https://github.com/samvera/hyrax/blob/main/app/controllers/concerns/hyrax/singular_subresource_controller.rb#L14
> 
> WorkRelation is a subclass of ActiveFedora::Relation:
> [5:33](https://samvera.slack.com/archives/CDRHAMUTV/p1647552796997939)
> https://github.com/samvera/hyrax/blob/f050d8f7f644321f1524af26c1adc18cc0eff0f8/app/search_builders/hyrax/abstract_type_relation.rb#L3

Tracking in GitHub, this line came over with Sufia.  It was used to find the work through an ActiveFedora relationship.  This no longer happens.  Only `@presenter` is referenced in the [view](https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/views/hyrax/citations/work.html.erb).

Reply:
Digging a little deeper, the part of SingularSubresourceController that is used is `load_and_authorize_resource` for both work and file.  

### With SingularSubresourceController
If the user doesn't have access to the work or file...
* they are forwarded to Home and shown a message about not having access

### Without SingularSubresourceController
If the user doesn't have access to the work or file...
* they stay on the citations page and are shown a message about the work being private

Reply:
@elrayle For an user that is not logged in, the behavior is the same as what is is tested in the controller spec:
https://github.com/samvera/hyrax/blob/main/spec/controllers/hyrax/citations_controller_spec.rb#L27-L28

A logged-in user without permission has the same experience you mention above, but this is not tested in the specs. Is being re-directed to Home the preferred behavior?

Reply:
Pushed an update to redirect to the home page.